src/open_inwoner/openzaak/api_models.py

```diff
@@ -366,11 +366,11 @@ def process_data(self) -> dict:
 
 
 @dataclass
-class OpenTask(Model):
+class OpenstaandeTaak(Model):
     url: str
     uuid: str
     identificatie: str
     naam: str
-    startdatum: date
+    startdatum: Optional[date]
```

Comment:
Let's use newPython:


```suggestion
    startdatum: date | None
```

Reply:
The `zgw_consumers` library can't handle the `date | None` union syntax, it raises `TypeError` for this.